controller/openebs/common.go

```diff
@@ -237,6 +237,28 @@ func (p *Planner) removeDisabledManifests() error {
 		delete(p.ComponentManifests, types.CStorAdmissionServerManifestKey)
 	}
 
+	if *p.ObservedOpenEBS.Spec.MayastorConfig.Moac.Enabled == false &&
+		*p.ObservedOpenEBS.Spec.MayastorConfig.Mayastor.Enabled == false {
+		delete(p.ComponentManifests, types.MoacSAManifestKey)
+		delete(p.ComponentManifests, types.MoacClusterRoleManifestKey)
+		delete(p.ComponentManifests, types.MoacClusterRoleBindingManifestKey)
+		delete(p.ComponentManifests, types.MoacDeploymentManifestKey)
+		delete(p.ComponentManifests, types.MoacServiceManifestKey)
+		delete(p.ComponentManifests, types.MayastorDaemonsetManifestKey)
+	}
```

Comment:
what is the point of this "if" block above? It seems useless as the two if conditions below do basically the same thing.

Reply:
Yes, correct. I have updated the same. Thanks.